codahale/sneaker
# Remove vendor contents from source control

Don't need to track these, can rely on package manager to pull them down (looks like I accidentally checked these in a few commits ago).

Also, these files broke `go get` (fixes #29).

Test Plan:
- verify `make test` fails by default (no checked in deps)
- verify `govendor sync && make test` succeeds
- verify `go get -d -u github.com/dstnbrkr/sneaker` succeeds
